docs/changehistory/NextVersion.md

```diff
@@ -120,6 +120,14 @@ Several changes were made in the @bentley/ui-* packages.
 Some components in @bentley/ui-core were deprecated in favor of components in @itwinui-react.
 A few constructs were deprecated in @bentley/ui-core package with alternatives elsewhere.
 
+### @bentley/extension-cli
+
+The cli tool has been deprecates due to an impending change of Extensions and the Extension Service. Please continue to use the 2.x version if you still require publishing Extensions.
```

Comment:
deprecates => deprecated

Reply:
Thank you. I guess spell checkers can't spot grammar mistakes